packages/manager/modules/pci/src/projects/project/instances/instances.service.js

```diff
@@ -206,6 +206,18 @@ export default class PciProjectInstanceService {
     );
   }
 
+  shelve(projectId, { id: instanceId }) {
```

Comment:
You're passing an object param, which is destructured to use the object's id => just pass the id no ?

Reply:
Hello @JeremyDec ,
I think there is a lot of service in public-cloud where we call service method with an entire object and then destructuring to use only some properties. 

IMHO I think it's more easy and split the responsability: 
Here the service allow to shelve an instance, but when you use the service you don't have to think about the implementation behind the service `shelve` method (maybe it uses only the `id`, maybe it uses others properties...).

Reply:
seen together, it is ok with this way of passing parameters to this function